manifests/params.pp

```diff
@@ -109,6 +109,7 @@
   $manage_firewall                          = false
   $manage_apt                               = true
   $repo_location                            = ''
+  $unsupported_repo_location                = ''
```

Comment:
Can you please default to `undef`? Thats the new prefered default for strings (take a look at our review guidelines at https://voxpupuli.org/docs/#reviewing-a-module-pr )

Reply:
Absolutely!  I just mirrored the one above it in case that was the expected style for this project.  =)  Would you like me to adjust the previous one (repo_location) while I'm at it as well?

Reply:
Yes, you can change that one as well